src/services/analytics/events/addressBook.ts

```diff
@@ -2,44 +2,46 @@ import { GTM_EVENT } from '@/services/analytics/analytics'
 
 const ADDRESS_BOOK_CATEGORY = 'address-book'
 
+const event = GTM_EVENT.CLICK
+
 export const ADDRESS_BOOK_EVENTS = {
   EXPORT: {
-    event: GTM_EVENT.CLICK,
+    event,
```

Comment:
I meant in the function that tracks. This param should be optional.

Reply:
The types do not allow it.

Reply:
Then we need to fix the types.

Reply:
`trackEvent` would otherwise allow objects sans `event`.

Reply:
Yes, that's the point.